catalogs/catalog-lakehouse-paimon/src/main/java/org/apache/gravitino/catalog/lakehouse/paimon/filesystem/FileSystemType.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.gravitino.catalog.lakehouse.paimon.filesystem;
+
+public enum FileSystemType {
+  LOCAL_FILE,
+  HDFS,
+  S3;
+
+  public static FileSystemType fromString(String type) {
+    for (FileSystemType fileSystemType : FileSystemType.values()) {
+      if (fileSystemType.name().equalsIgnoreCase(type)) {
+        return fileSystemType;
+      }
+    }
+
+    throw new IllegalArgumentException("Unsupported file system type: " + type);
+  }
+
+  public static FileSystemType fromStoragePath(String storagePath) {
+    if (storagePath.startsWith("s3://")) {
```

Comment:
s3a?

Reply:
let me check it for Paimon, the vlaue is `s3://` in the example showed in https://paimon.apache.org/docs/0.8/filesystems/s3/

Reply:
Is the value `s3a` for Iceberg S3 storage?